adafruit_fakerequests.py

```diff
@@ -30,16 +30,16 @@
 class Fake_Requests:
     """For faking 'requests' using a local file instead of the network."""
 
-    def __init__(self, filename):
+    def __init__(self, filename: str) -> None:
         self._filename = filename
 
-    def json(self):
+    def json(self) -> str:
```

Comment:
This will be a Python object return, but it's anyone's guess what exactly!  For that reason, add the typical `typing` import block, import `Any` from it, and make this return `Any`.

Reply:
lol - was not expecting feedback before next week.....how did you do this and the other on CP Day?

Reply:
We're doing a mini-sprint, so I'm reviewing with a bunch of people at lightning speed! :laughing: